### PR TITLE
Add missing checked-in generated code for vectorAsIterable

### PR DESCRIPTION
**Pull request by tristanlabelle**
Because I implemented https://github.com/thebrowsercompany/swift-winrt/pull/158 in `github.dev`.

### CODE REVIEW

File: tests/test_component/Sources/test_component/test_component+ABI.swift [hunk 1]
```diff
@@ -1086,6 +1086,17 @@ public enum __ABI_test_component {
             }
         }
 
+        internal func VectorAsIterableImpl(_ value: test_component.AnyIVector<String>?) throws -> test_component.AnyIIterable<String>? {
+            let (result) = try ComPtrs.initialize { resultAbi in
```
Comment (jeffdav) [inline tests/test_component/Sources/test_component/test_component+ABI.swift:1090]: Why does result have parens?

**Review by jeffdav — APPROVED** [review]

File: tests/test_component/Sources/test_component/test_component+ABI.swift (hunk 1 above)
Comment (jeffdav) [reply tests/test_component/Sources/test_component/test_component+ABI.swift:1090]: OK.  I would have thought that would make it a tuple of one?  I dunno.  If it works leave it.

Comment (tristanlabelle) [reply tests/test_component/Sources/test_component/test_component+ABI.swift:1090]: I think a tuple of one and a scalar are the same thing.

Comment (tristanlabelle) [reply tests/test_component/Sources/test_component/test_component+ABI.swift:1090]: unless they are not and the `let (x)` syntax deconstructs a tuple of one into a scalar declaration

**Pull request merged**